<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdicionarWindow</class>
 <widget class="QMainWindow" name="AdicionarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Adicionar item</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formQtd" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelQtd">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantidade:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxQtd">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999999</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="formNome" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelNome">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="lineEditID">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999999</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="formCategoria" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEditCategoria">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelCategoria">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Categoria:</string>
     </property>
    </widget>
    <zorder>labelCategoria</zorder>
    <zorder>lineEditCategoria</zorder>
   </widget>
   <widget class="QWidget" name="formPreco" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelPreco">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pre√ßo:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>166</width>
      <height>24</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Arial</family>
     <fontweight>Thin</fontweight>
    </font>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdicionarWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>customSlot()</slot>
 </slots>
</ui>
